onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorGemm.cpp

```diff
@@ -10,12 +10,13 @@ class DmlOperatorGemm : public DmlOperator, public GemmHelper
 {
 public:
     DmlOperatorGemm(const MLOperatorKernelCreationContext& kernelInfo)
-        :   DmlOperator(kernelInfo), 
+        :   DmlOperator(kernelInfo),
             GemmHelper(kernelInfo, kernelInfo.GetTensorShapeDescription())
     {
         ML_CHECK_VALID_ARGUMENT(kernelInfo.GetInputCount() >= 2);
         ML_CHECK_VALID_ARGUMENT(kernelInfo.GetOutputCount() == 1);
-        DmlOperator::Initialize(kernelInfo);
+        auto kernelInputIndices = std::vector<std::optional<uint32_t>> { 0, 1, 2 };
```

Comment:
Thank you for pointing this out. I think we need to enhance our ONNXConformance Test to test each operator with/without the graph. Right now OnnxConformance test, executes the GEMM as a graph node. And the Compute method is different for an individual operator and a DML Graph. I will discuss this with the team in more detail.